samples/matter/window_covering/boards/nrf54l15pdk_nrf54l15_cpuapp_0_3_0 _release.overlay

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
This is a white symbol between 0_3_0 and  _release in the filename.

Reply:
Ohhh, falcon eye! Thanks